src/app/user/adapter/UserSequelizeAdapter.js

```diff
@@ -9,6 +9,7 @@ const generateSalt = require('./../utils/generateSalt');
 const { v4: uuid } = require('uuid');
 const { promisify } = require('util');
 const crypto = require('crypto');
+const config = require('./../../../infrastructure/config');
```

Comment:
Can we remove line 12 - `config` is never used

Reply:
Removed unused import @ipredoi